### PR TITLE
New Page: aria-label

### PR DESCRIPTION
**Pull request by estelle**
- [X] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by ericwbailey — CHANGES_REQUESTED** [review]
Left some changes. One big picture change I'd like to see is a note indicating that `aria-label` is intended for use on interactive elements, or elements made to be interactive via other ARIA declarations.

I see a lot of developers using `aria-label` like a tooltip to provide additional instructions or clarify about use on UI. When it is placed on an non-interactive element, typically a `div` it may not be read, or may confuse a user as it is a non interactive element that acts like an interactive one. Int he case of a tooltip, it may override or alter the accessible name, making it verbose, confusing, and possibly making it impossible to know how to activate via voice control. Finally, tooltip-esque control hints given via `aria-label` [muck with users' expectations and may give bad or irrelevant instructions](https://adrianroselli.com/2019/10/stop-giving-control-hints-to-screen-readers.html).

File: files/en-us/web/accessibility/aria/attributes/aria-label/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+The `aria-label` attribute defines a string value that labels the current element.
+
+## Description
+
+Sometimes, like with SVG or [icon fonts (which you shouldn't be using)](https://www.youtube.com/watch?v=9xXBYcWgCHA), the default [accessible name](/en-US/docs/Glossary/Accessible_Name) of an object is missing or isn't really what the object is meant to represent, and there is no content visible in the DOM that can be associated with the object to give it meaning. In cases where an element has no accessible name or an accessible name that isn't accurate, and there is no content available and visible in the DOM to label the element, the `aria-label` attribute can be used to define a string that labels the element on which it is set, providing it with it's accessible name. 
+
+```html
+<button aria-label="Close" onclick="myDialog.close()">X</button>
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/accessibility/aria/attributes/aria-label/index.md:20]: nice!

note to self: edit word "hidden"

**Comment by estelle** [comment]
The tooltip stuff should go in tooltip role, but I added several lines about interactive only and don't provide info to AT and not all users.

**Review by ericwbailey — APPROVED** [review]
Nice! Thank you for the changes.

**Pull request merged**